### DISCUSSION TITLE
Curso Dominios Ricos (Dúvida)

### DISCUSSION THREAD

**Discussion by fabioborgesconttrotech** [balta-io/forum#544] [Tira Dúvidas] [answered]
Balta, buenos dias....
uma dúvida de leigo mesmo.... 
Um ERP grande no projeto, seguindo o modelo do curso,  seria apenas um projeto Domain e um Shared, ou seja, o ERP teria uma lib de domain e uma shared seria isso mesmo?
Obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @fabioborgesconttrotech, como vai?

Depende do "tamanho"... pode ser N projetos dentro de uma solução, pode ser N pacotes ou N serviços.

Não vejo problema em ser apenas um Domain, mas normalmente para projetos grandes separamos em múltiplos projetos (Pelo menos) para organizar melhor.

**Comment by fabioborgesconttrotech** [comment]
valeu fera.
